packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -442,6 +442,10 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
 
   private _renderNormalItem(item: IContextualMenuItem, classNames: IMenuItemClassNames, index: number, focusableElementIndex: number, totalItemCount: number, hasCheckmarks: boolean, hasIcons: boolean): React.ReactNode {
     if (item.onRender) {
+      item.positionInSet = focusableElementIndex;
+      item.setSize = totalItemCount;
+      item.hasCheckMarks = hasCheckmarks;
+      item.hasIcon = hasIcons;
```

Comment:
Do we currently have a need have a need for the hasCheckmarks or hasIcons? If not, it should only be added once it is needed

Reply:
No we don't. They have been removed.